01 - Community/Obsidian Roundup/2022-06-18 Breaking Changes & Dataview API Upgrades.md

```diff
@@ -0,0 +1,101 @@
+-   [Another Quick Switcher plugin v4.14.0](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/4.14.0) now has numbers for quicker result selection and added a "hide gutter icons" option.
+-   [Obsidian Rewarder](https://github.com/Gnopps/obsidian-rewarder) allows users to ignore unwanted rewards, add images to rewards, and log all completed tasks to a daily note.
+-   [Tasks Plugin 1.8.0](https://github.com/obsidian-tasks-group/obsidian-tasks/releases/tag/1.8.0) got a dependencies update, so make sure to restart Obsidian after you update (This is a good habit to be in anyway, honestly), a [sample vault](https://github.com/obsidian-tasks-group/obsidian-tasks/tree/main/resources/sample_vaults/Tasks-Demo), and new date abbreviations. Here's a [quick reference](https://obsidian-tasks-group.github.io/obsidian-tasks/quick-reference/) for Tasks and a [discussion about workarounds for OR combinations of filters](https://github.com/obsidian-tasks-group/obsidian-tasks/discussions/746).
+-   [Media DB Plugin - 0.3.0](https://github.com/mProjectsCode/obsidian-media-db-plugin) added bulk importing, the ability to select multiple results at once, and a bugfix.
+-   Obsidian Social (Sekund), which lets users share notes with other people and get comments, [now works on mobile](https://forum.obsidian.md/t/obsidian-social-sekund-now-works-on-mobile/39046?u=ryanjamurphy).
+-   [Smarter Markdown Hotkeys](https://obsidian.md/plugins?id=obsidian-smarter-md-hotkeys) allows you to increase heading levels, toggle readable line length, toggle line numbers, and `toggle Between CASES`.
+-   [Theme Design Utilities](https://obsidian.md/plugins?id=obsidian-theme-design-utilities) makes it easier to toggle between your installed themes.
+
+### Betas
+
+_Note: these plugins have not yet been submitted for code review, and are being made available primarily for testing purposes._
+
+-   [Obsidian Translate](https://github.com/Fevol/obsidian-translate/releases/tag/1.0.1.1) integrates with 5 different translation APIs to translate text after automatically identifying the text's language. There's also a neat custom view.
+-   The [Meta Bind](https://github.com/mProjectsCode/obsidian-meta-bind-plugin) plugin (which I almost typo'd as "mind bend," whups) can create input fields inside notes and bind them to metadata fields.
+
+### For Developers
+
+-   CodeMirror 6.0 is finally stable. That means we’ll be migrating from CM6 v0.19 (aka v6.0-beta) to v6.0 soon. The migration will have some breaking changes, but Licat helpfully put together this [migration guide](https://forum.obsidian.md/t/plugin-developers-codemirror-6-migration-guide-for-v6-0/38947).
+
```

Comment:
Another extraneous newline, will need to check upstream to see if it's present there as well.

Reply:
Thanks. I'm not seeing the extra newline?

Reply:
Line 58 of this file is showing up as a blank line for me. Looks fine on the Roundup. Let me check the RSS.

Reply:
It would appear that this no longer exists in the Roundup RSS feed.

Reply:
> It would appear that this no longer exists in the Roundup RSS feed.

Yes, it was fortunate that I had used the RSS plugin to download them before they expired from the RSS feed...

I'm not sure how you are checking them, but if they look OK in Obsidian they will look OK in Obsidian Publish.